cores.json

```diff
@@ -0,0 +1,466 @@
+[
+    {
+        "name": "mame2003-libretro",
+        "makeoptions": {
+            "buildpath": "./",
+            "makescript": "Makefile",
+            "arguments": []
+        },
+        "options": {
```

Comment:
How do we plan on passing these to EmulatorJS? Do you think it would be possible to compress a `default-options.json` (or something) file in the core too? Though this would require editing the dist-cores.sh script in RetroArch

Reply:
This stanza exists to tell EJS to create the .opt files (in the EJS project, see GameManager.js:33). This effectively replaces lines like this: ``this.writeFile("/home/web_user/retroarch/userdata/config/Beetle PSX HW/Beetle PSX HW.opt", 'beetle_psx_hw_renderer = "software"\n');``

When EJS starts, it'll check for the existance of data/cores/cores.json and load it into memory, then search for the selected core - I would then suggest storing the selected core in a variable to make lookups easier.